ci/Jenkinsfile

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env groovy
+library 'status-jenkins-lib@v1.7.13'
+
+pipeline {
+    agent {
+        label 'linux && x86_64 && qt-5.15.2'
+    }
+
+    environment {
+        SQUISH_DIR = "/opt/squish-runner-7.1-20230222-1555"
+        PYTHONPATH = "${SQUISH_DIR}/lib:${SQUISH_DIR}/lib/python:${PYTHONPATH}"
+        LD_LIBRARY_PATH = "${SQUISH_DIR}/lib:${SQUISH_DIR}/python3/lib:${LD_LIBRARY_PATH}"
+    }
+
+    parameters {
+        string(
+          name: 'BRANCH',
+          description: 'Paste a branch name from test repository.',
+          defaultValue: 'master'
+        )
+        string(
+          name: 'BUILD',
+          description: 'Paste a number of build that you want to test.',
+          defaultValue: 'PR-'
+        )
+        string(
+          name: 'FILTER',
+          description: 'Paste "-k" with test name/part of test name to run specific test or/and "-m" with tag to run specific scope of tests',
+          defaultValue: ''
+        )
+
+        string(
+          name: 'TEST_RUN',
```

Comment:
what is that? Test run should be created automatically when tests are launched. How to use that?

Reply:
description: 'Test run ID in Test Rail'
It is a field where we need to paste the Test Run ID from TestRail
It is not created automatically
How to use:
1) Create a Test Run in the TestRail with test cases that should be executed to test PR
2) Start Autotests, with TesRun Id in TEST_RUN field
3) When autotests are finished, execute test cases that were not automated or failed in TestRun manually
4) Close PR or Move it back in development